plots/mock/README.md

```diff
@@ -0,0 +1,16 @@
+Is there a relationship between predator size (mass and length) and prey size (mass and length)?
+Caption: 
```

Comment:
We might want to add a figure number to each figure so that it's easier to tell which caption goes with which figure